### DISCUSSION TITLE
Append not working?

### DISCUSSION THREAD

**Discussion by ruiestrelinha** [biati-digital/glightbox#368] [Q&A]
Update: Found the answer, my mistake sorry! Can delete this entry, thanks!
---

Hi,

Sorry for the dumb question but I tried everything I know and can't find the reason to the glightbox is not working.
Issue:
If I write directely a link in the html the glightbox trigger works fine but if add the link to the html dynamically with js (append or innerHTML methods) than the trigger doesn't work and apparently the code seems equal.
Any sugestions to solve this?

Thank you all

**Comment by gingerchew** [comment]
Well don't leave us hanging! :)

If you want, you can share the solution you found, I'm sure it would help others who run into a similar issue.
